docusaurus/docs/React/components/core-components/thread-list.mdx

```diff
@@ -3,9 +3,9 @@ id: thread-list
 title: ThreadList
 ---
 
-`ThreadList` is a component which renders individual thread instances ([`Thread`](https://github.com/GetStream/stream-chat-js/blob/master/src/thread.ts)) stored within `ThreadManager`. It handles pagination triggers and virtualization through the help of the [Virtuoso](https://virtuoso.dev) virtualized list component. The rest of the business logic lives within ThreadManager and Thread classes. ThreadManager instance gets activated whenever ThreadList renders - activation is necessary as it tells the SDK that user "sees" this list and can update state accordingly whenever appropriate events arrive.
+`ThreadList` is a component which renders individual thread instances ([`Thread`](https://github.com/GetStream/stream-chat-js/blob/master/src/thread.ts)) stored within [`ThreadManager`](https://github.com/GetStream/stream-chat-js/blob/master/src/thread_manager.ts). It handles pagination triggers and virtualization through the help of the [Virtuoso](https://virtuoso.dev) virtualized list component. The rest of the business logic lives within ThreadManager and Thread classes. ThreadManager instance gets activated whenever ThreadList renders - activation is necessary as it tells the SDK that user "sees" this list and can update state accordingly whenever appropriate events arrive.
```

Comment:
I think it's better to explain `ThreadList` from the user standpoint. In most cases users don't know about ThreadManager and thread instances. So a good introduction would be

> `ThreadList` is a component that displays a list of threads where current user is a participant (user either started a thread or replied in a thread).

The same goes for `Virtuoso` (from the user standpoint, there is no Virtuoso, just `VirtualizedMessageList`), activation (it happens automatically, so why mention it) etc.